test/jdk/java/awt/Frame/MaximizedToIconified/MaximizedToIconified.java

```diff
@@ -114,6 +116,7 @@ private static void doTest() {
         //    because Toolkit.isFrameStateSupported() method reports these states
         //    as not supported. And such states will simply be skipped.
         examineStates(new int[] {Frame.MAXIMIZED_BOTH, Frame.ICONIFIED, Frame.NORMAL});
+        System.out.println("------");
         examineStates(new int[] {Frame.ICONIFIED, Frame.MAXIMIZED_BOTH, Frame.NORMAL});
 
```

Comment:
@alisenchung Now that the `lastFrameState` is fixed, I would suggest adding the cases mentioned [here](https://github.com/openjdk/jdk/pull/14226/files#r1237259547) too, to make the test comprehensive.